Loop Status Extension/NSBundle+LoopStatusExtension.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  NSBundle+LoopStatusExtension.swift
+//  Loop
+//
+//  Created by Bharat Mediratta on 12/7/16.
+//  Copyright © 2016 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+extension Bundle {
+    static var appGroupSuiteName: String {
```

Comment:
You can share this code between targets now (and it can go in the follow-up framework refactor later)